tests/test.sh

```diff
@@ -121,6 +121,22 @@ fi
 
 check_valgrind_log "bad words"
 
+# Tests good words' root
+if test -f $TESTDIR/$NAME.root; then
+    hunspell $* -d $TESTDIR/$NAME <$TESTDIR/$NAME.good | grep -a '^+ ' | \
```

Comment:
Could you add a comment what's the intended effect of "grep" and "sed" calls here? It's not obvious to me why are they needed.